app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -537,7 +550,7 @@ protected void onPostExecute(DBReader.NavDrawerData result) {
                 String lastFragment = getLastNavFragment();
                 if(!ArrayUtils.contains(NAV_DRAWER_TAGS, lastFragment)) {
                     long feedId = Long.valueOf(lastFragment);
-                    loadFeedFragmentById(feedId);
+                    //loadFeedFragmentById(feedId);
```

Comment:
The idea is to refactor the whole junk as we are not showing the feed list in the navigation bar.  Since, a lot of changes are involved, I am reverting back the code.  But, I believe, In future we could just refresh the count of "Queue" and "New" instead of the whole feed list.